lib/Form/Processor/ContentTypeFormProcessor.php

```diff
@@ -116,5 +133,20 @@ public function processRemoveContentTypeDraft(FormActionEvent $event)
                 new RedirectResponse($this->router->generate($this->options['redirectRouteAfterPublish']))
             );
         }
+
+        $this->addNotification('content_type.notification.draft_removed');
+    }
+
+    /**
+     * Pushes a flash notification to session.
+     *
+     * @param string $message
+     */
+    private function addNotification($message, array $params = [])
```

Comment:
I don't really see the need for a service here since the session is already a service we use for that.
As for the translator I'd say the same. Furthermore, the *context* (here `ezrepoforms_content_type`) will change, so it won't be that generic.

That's mainly why I chose a private method over a dedicated service.